src/delfino/models/command_package.py

```diff
@@ -0,0 +1,22 @@
+from importlib.resources import Package
+
+from pydantic import BaseModel
+
+
+class CommandPackage(BaseModel):
+    """Hold importlib.resources.Package that include commands and metadata for delfino.
+
+    Attributes:
+        package (Package): A package that include commands.
+        required (bool): One ore more commands from this package is required.
+        new_name (str): Set when ``package`` is a legacy name and ``new_name`` should be used instead.
+        plugin_name (str): Name of the plugin.
```

Comment:
The attributes are already type annotated so no need for another annotation in the docstring. It will only lead to lying about the code when someone in the future changes the type annotation and forgets to update the docstring. Check out other places in your PR for these instances.

```suggestion
        package: A package that include commands.
        required: One ore more commands from this package is required.
        new_name: Set when ``package`` is a legacy name and ``new_name`` should be used instead.
        plugin_name: Name of the plugin.
```